src/components/metadata/styles/metadata.scss

```diff
@@ -193,18 +193,13 @@
 $list-inline-spacing: 0.2em;
 
 .pipeline-metadata__value-list--inline {
-  margin: 0 (-$list-inline-spacing * 1.5);
   padding: 0;
+  line-height: 1.5em;
```

Comment:
I think we need a `margin: 0;` here. The distance between `Transcoded Types:` and the line below is greater than the distance between `Original Type:` and the line below:

<img width="281" alt="image" src="https://user-images.githubusercontent.com/16869061/216073931-226822cc-7ba7-4e07-bafb-4cb43dd649fe.png">